### PR TITLE
Dash quality

### PR DESCRIPTION
**Pull request by yaniv-li**

### CODE REVIEW

File: src/plugins/videojs-http-source-selector/plugin.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import videojs from 'video.js';
+
+import SourceMenuButton from './components/SourceMenuButton';
+import SourceMenuItem from './components/SourceMenuItem';
+import qualitySelector from '../../components/qualitySelector/qualitySelector';
```
Comment (tsi) [inline src/plugins/videojs-http-source-selector/plugin.js:5]: @yaniv-li this seems to be un-used

File: src/plugins/videojs-http-source-selector/plugin.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    return false;
+  }
+
+  /**
+  *
+  * We have to wait for the manifest to load before we can scan renditions for resolutions/bitrates to populate selections
+  *
+  **/
+  player.on(['loadedmetadata'], function(e) {
+    let qualityLevels = player.qualityLevels();
```
Comment (tsi) [inline src/plugins/videojs-http-source-selector/plugin.js:44]: @yaniv-li this seems to be un-used

File: src/components/qualitySelector/qualitySelector.js [hunk 3]
```diff
@@ -98,6 +124,18 @@ const qualitySelector = {
     if (sourceMenuButton) {
       const qualityLevels = player.qualityLevels();
       if (qualityLevels && qualityLevels.length > 1) {
+        let levels = qualityLevels.levels_.filter((q) => q.enabled);
+        if (levels.length === 1) {
+          let idx = qualityLevels.levels_.findIndex(l => l.id === levels[0].id);
+          sourceMenuButton.children()[1].children()[idx].selected(true);
```
Comment (tsi) [inline src/components/qualitySelector/qualitySelector.js:130]: @yaniv-li following our previous discussion 
for marking selected option, see [what they did here](https://github.com/cloudinary/cloudinary-video-player/pull/216/files#diff-3bba335ded3dcda3c175fc8e6c2dd199bbaba7641b11ca9dd014ec0a2090c131R18):
```
let levels = this.player().qualityLevels();
levels[i].enabled = true;
```

File: src/video-player.js [hunk 4]
```diff
@@ -360,7 +360,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
     this.initQualitySelector = () => {
       if (_options.qualitySelector !== false) {
-        this.videojs.httpSourceSelector({ default: 'high' });
+        this.videojs.httpSourceSelector({ default: 'Auto' });
```
Comment (tsi) [inline src/video-player.js:363]: @yaniv-li should this be 'auto' or 'Auto'? [see readme](https://www.npmjs.com/package/videojs-http-source-selector#auto)
either way, I don't see any change when playing with this value

**Review by tsi — COMMENTED** [review]
@yaniv-li code looks good and I tested this locally and it works great! for both HLS & DASH

few notes though:
pls see linter warnings about `no-unused-vars`
do we really need both Component and Plugin? can we unite them both into the plugin?
Also, there's no need for the `videojs-` in the plugin name `videojs-http-source-selector` :)

File: src/components/qualitySelector/qualitySelector.js (hunk 3 above)
Comment (yaniv-li) [reply src/components/qualitySelector/qualitySelector.js:130]: it should be enabled this is just to make sure the visual part is ok

File: src/video-player.js (hunk 4 above)
Comment (yaniv-li) [reply src/video-player.js:363]: I think we should re-implement this according to what our design

Comment (maoznir) [reply src/video-player.js:363]: show be lower case i think

**Review by maoznir — APPROVED** [review]

**Pull request merged**